### DISCUSSION TITLE
UserManager가 해야할 일들이 뭐가 있을까?

### DISCUSSION THREAD

**Discussion by sseungmn** [boostcampwm-2022/iOS09-burstcamp#172] [General]
모듈화 초안
<img width="719" alt="2022-12-05_11-35-11" src="https://user-images.githubusercontent.com/46219689/205536944-324d20d9-8a7a-4799-b940-d3913f8afbf5.png">

**Comment by guesswb** [comment]
Auth에서 KeyChain 접근 안하고 있는데 어떤기능을 염두해두고 선을 그어놓으신 건가요?
